<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;幼圆&quot;; </string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>30</y>
      <width>481</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="roomNumLabel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>房间号</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>181</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>温度信息</string>
     </property>
     <widget class="QLabel" name="temNumLabel">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>54</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>99.9</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="ActualTemLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>当前温度：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="goalTemLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>目标温度：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>99</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>37</width>
        <height>18</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="arrowType">
       <enum>Qt::UpArrow</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>37</width>
        <height>18</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="arrowType">
       <enum>Qt::DownArrow</enum>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>181</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>工作状态</string>
     </property>
     <widget class="QLabel" name="workType">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>54</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>俩字</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="ActualWindLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>当前风速：</string>
      </property>
     </widget>
     <widget class="QLabel" name="windNumLabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>中风</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>37</width>
        <height>18</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonIconOnly</enum>
      </property>
      <property name="arrowType">
       <enum>Qt::UpArrow</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>37</width>
        <height>18</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="arrowType">
       <enum>Qt::DownArrow</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>54</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>工作模式：</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>120</y>
       <width>171</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>客户信息</string>
     </property>
     <widget class="QLabel" name="payNumLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>姓名：</string>
      </property>
     </widget>
     <widget class="QLabel" name="payLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>费用：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="switchButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>260</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>关机</string>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>54</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>等待链接</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
